pandas/tests/extension/test_arrow.py

```diff
@@ -374,21 +370,42 @@ def test_accumulate_series_raises(
     @pytest.mark.parametrize("skipna", [True, False])
     def test_accumulate_series(self, data, all_numeric_accumulations, skipna, request):
         pa_type = data.dtype.pyarrow_dtype
+        op_name = all_numeric_accumulations
+        ser = pd.Series(data)
+
+        if pa.types.is_string(pa_type) or pa.types.is_binary(pa_type):
```

Comment:
Could you change these to use pytest skips with a message that the type & accumulator isn't supported?